src/bot.ts

```diff
@@ -231,11 +231,25 @@ export class Bot<
      * Initializes the bot, i.e. fetches information about the bot itself. This
      * method is called automatically, you usually don't have to call it
      * manually.
+     *
+     * This method allows you to pass `{ unreliable: true }` which will make the
```

Comment:
I think the name `unreliable` is horribly self documenting. I read the jsdoc text and its still not fully clear to me what this does. Then I looked into the code and I understood what it does but that shouldn't be needed.

I assume a name like `disableInternalErrorhandling`, `disableInternalRetries` or `useExternalErrorhandling` would be more helpful to understand what it does or what it is meant to do. Especially if its not meant for everyone, just use a long variable name there which describes more than a single adverb.